UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -10,9 +10,105 @@ import UIKit
 /// ViewController
 class ViewController: UIViewController {
```

Comment:
```suggestion
class StartViewController: UIViewController {
```
и описание не говорящее, чем занимается этот контроллер

Reply:
Исправил замечание